### PR TITLE
Recipe App Group Project

### PR DESCRIPTION
**Pull request by mutinhiri**
In this Project @mutinhiri , @Emmy-github-webdev  and @nicupop729 we implemented:

- A login page.
- A registration page.
- A food list (with all CRUD implementation, except for 'update').
- A recipes list (with all CRUD implementation, except for 'update').
- A list of all public recipes from other users with their names and total prices.
- Inventories list (with all CRUD implementation, except for 'update').

### CODE REVIEW

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- Kudos on the good README
- Kudos on testing your application
- No linter errors


## Required Changes ♻️
_Check the comments under the review._




### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: app/views/recipe_foods/_recipe_food.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  <p>
+    <strong>Recipe:</strong>
+    <%= recipe_food.recipe_id %>
+  </p>
+
+  <p>
+    <strong>Food:</strong>
+    <%= recipe_food.food_id %>
+  </p>
+</div>
```
[the remark this answers is not in the document]
Comment (Emmy-github-webdev) [reply app/views/recipe_foods/_recipe_food.html.erb:16]: The issue is fixed. Thank you @DeliceLydia

File: app/views/foods/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+          <tr>
+            <td><%= index+=1 %></td>
+            <td><%= food.name %></td>
+            <td><%= food.measurement_unit %></td>
+            <td>$<%= food.price %></td>
+            <td>
+            <%= button_to "Delete", food, method: :delete, class: 'delete-btn btn' %>
+            </td>
+          </tr>
+        <% end %>
```
[the remark this answers is not in the document]
Comment (Emmy-github-webdev) [reply app/views/foods/index.html.erb:29]: Thank you @DeliceLydia

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Emmy-github-webdev** [comment]
Thanks @elisha2kyakpo1

**Pull request merged**